internals/secrethub/read.go

```diff
@@ -60,6 +71,10 @@ func (cmd *ReadCommand) Run() error {
 		return err
 	}
 
+	if cmd.useClipboard && cmd.outFile != "" {
```

Comment:
Why is this mutually exclusive?

Reply:
Hmm, you're right. I've updated it too allow both to be used together.